app/src/main/java/com/example/courses/presentation/ui/MainFragment.kt

```diff
@@ -32,67 +31,59 @@ class MainFragment : Fragment(), ItemClickListener {
-
-    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?,
-                              savedInstanceState: Bundle?): View {
+    private var adapter = PersonAdapter(mutableListOf())
+    private lateinit var person: Person
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View {
         val view = inflater.inflate(R.layout.main_fragment, container, false)
         return view
     }
 
     override fun onActivityCreated(savedInstanceState: Bundle?) {
         super.onActivityCreated(savedInstanceState)
         viewModel = ViewModelProvider(this).get(MainViewModel::class.java)
+        person = viewModel.take()
+        inputFirst.setText(person.name, TextView.BufferType.EDITABLE)
+        inputSecond.setText(person.rating.toString(), TextView.BufferType.EDITABLE)
         inputFirst.doAfterTextChanged {
             viewModel.first = it.toString()
         }
         inputSecond.doAfterTextChanged {
             viewModel.second = it.toString()
         }
         btncalculate.setOnClickListener {
-            val toast = Toast.makeText(requireContext(), "${viewModel.calculate()}", Toast.LENGTH_SHORT)
-            toast.show()
+            viewModel.creat()
+            viewModel.save()
         }
-        viewModel.getOperations().observe(viewLifecycleOwner, Observer {
+        viewModel.getPersons().observe(viewLifecycleOwner, Observer {
             adapter.setData(it)
         })
 
-        viewModel.calculationState.observe(viewLifecycleOwner, Observer {
-            when (it) {
-                CalculationState.Free -> btncalculate.isEnabled = true
-                CalculationState.Loading -> btncalculate.isEnabled = false
-                CalculationState.Result -> btncalculate.isEnabled = false
-            }
-
-            textState.text = getString(when (it) {
-                CalculationState.Free -> R.string.free
-                CalculationState.Loading -> R.string.loading
-                CalculationState.Result -> R.string.result
-            })
-        })
-
     }
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
-        super.onViewCreated(view, savedInstanceState)
         inputFirst = view.findViewById(R.id.edit_Text_First)
         inputSecond = view.findViewById(R.id.edit_Text_Second)
-        btncalculate = view.findViewById(R.id.calculate)
-        operation = view.findViewById(R.id.operation)
+        btncalculate = view.findViewById(R.id.creat)
+        persons = view.findViewById(R.id.persons)
         textState = view.findViewById(R.id.state_text)
-        operation.layoutManager = LinearLayoutManager(requireContext())
-        operation.adapter = adapter
+        persons.layoutManager = LinearLayoutManager(requireContext())
+        persons.adapter = adapter
         adapter.setListener(this)
+        super.onViewCreated(view, savedInstanceState)
```

Comment:
All calls of super methods of system life cycle should be the first call of a method. If you have a specific reason to move it to the second or the last string, you should add a comment with describing this logic.